### PR TITLE
UT-3642 Update FBX SDK Bindings

### PR DESCRIPTION
**Pull request by nat-w**
Update Autodesk FBX SDK bindings to 3.1.0-preview.2.
Update exporter version to 3.2.1-preview.1.
Update changelog for 3.2.1-preview.1.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,21 @@
...
+- Add the option to export FBX files outside of the Assets folder.
+
+### Changed
+- Changed label name in Autodesk® Maya® from Unity Fbx Namespace to Strip Specific Namespace.
+- Changed Export as FBX Linked Prefab to Export as FBX Prefab Variant.
+- Mesh instances are now exported as instances of a single mesh instead of exporting multiple, identical meshes.
+- Updated to latest com.autodesk.fbx (3.1.0-preview.2).
+
+### Fixed
+- No longer intiate export if no objects are selected in Autodesk® 3DS Max®.
```
Comment (lassond) [inline CHANGELOG.md:15]: little typo here (intiate->initiate)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,21 @@
...
+- Added a null check for bones, so export no longer fails if skeleton has missing bones.
```
Comment (lassond) [inline CHANGELOG.md:16]: if skeleton -> if a skeleton

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,21 @@
 # Changes in Fbx Exporter
 
+## [3.2.1-preview.1] - 2020-08-05
+### Added
+- Add an export option to preserve model import settings when overwriting an fbx file.
+- Add the option to export FBX files outside of the Assets folder.
+
+### Changed
+- Changed label name in Autodesk® Maya® from Unity Fbx Namespace to Strip Specific Namespace.
+- Changed Export as FBX Linked Prefab to Export as FBX Prefab Variant.
```
Comment (benoithudson) [inline CHANGELOG.md:10]: For these two, I think "Renamed" instead of Changed

Comment (nat-w) [reply CHANGELOG.md]: Done!

**Pull request merged**